frontend/src/metabase/components/form/FormWidget.jsx

```diff
@@ -56,7 +61,11 @@ const FormWidget = forwardRef(function FormWidget(
   ref,
 ) {
   const Widget = getWidgetComponent(formField);
-  return <Widget field={field} {...formField} {...props} ref={ref} />;
+  return (
+    <Suspense fallback={<LoadingAndErrorWrapper loading />}>
```

Comment:
Does this cause a content layout shift (a flicker)?
A recording of how it behaves now would be nice, or at least steps to verify that these widgets work.